--- s/build/ac-macros/neon.m4.org	2006-11-08 13:40:10.387076060 +0000
+++ s/build/ac-macros/neon.m4	2006-11-08 13:41:47.125555455 +0000
@@ -127,11 +127,13 @@
       NEON_VERSION=`$neon_config --version | sed -e 's/^neon //'`
       AC_MSG_RESULT([$NEON_VERSION])
 
-      if test -n "`echo \"$NEON_VERSION\" | grep '^0\.2[[56]]\.'`" ; then
+      n_1=`echo "$NEON_VERSION" | grep '^0\.2[[56]]\.'`
+      if test -n "$n_1" ; then
         AC_DEFINE_UNQUOTED([SVN_NEON_0_25], [1],
                            [Define to 1 if you have Neon 0.25 or later.])
       fi
-      if test -n "`echo \"$NEON_VERSION\" | grep '^0\.26\.'`" ; then
+      n_2=`echo "$NEON_VERSION" | grep '^0\.26\.'`
+      if test -n "$n_2" ; then
         AC_DEFINE_UNQUOTED([SVN_NEON_0_26], [1],
                            [Define to 1 if you have Neon 0.26 or later.])
       fi
--- subversion-1.4.3/configure.in~	2007-01-01 15:41:51.000000000 +0100
+++ subversion-1.4.3/configure.in	2007-02-27 13:56:56.084373983 +0100
@@ -69,7 +69,7 @@
 
 # Either a space-separated list of allowable Neon versions, or "any" to
 # mean allow anything.
-NEON_ALLOWED_LIST="0.24.7 0.25.0 0.25.1 0.25.2 0.25.3 0.25.4 0.25.5 0.26.0 0.26.1 0.26.2"
+NEON_ALLOWED_LIST="0.24.7 0.25.0 0.25.1 0.25.2 0.25.3 0.25.4 0.25.5 0.26.0 0.26.1 0.26.2 0.26.3"
 NEON_LATEST_WORKING_VER=0.25.5
 NEON_URL="http://www.webdav.org/neon/neon-${NEON_LATEST_WORKING_VER}.tar.gz"
 dnl You can skip the neon version check only if you know what you are doing 
