diff -urN subversion-20020412.org/ac-helpers/berkeley-db.m4 subversion-20020412/ac-helpers/berkeley-db.m4
--- subversion-20020412.org/ac-helpers/berkeley-db.m4	Fri Apr 12 20:50:00 2002
+++ subversion-20020412/ac-helpers/berkeley-db.m4	Fri Apr 12 20:51:06 2002
@@ -102,7 +102,7 @@
     if test "$enable_shared" = "yes"; then
         SVN_DB_LIBS="$dbdir/libdb-4.0.la"
     else
-        SVN_DB_LIBS="-L$dbdir -ldb"
+        SVN_DB_LIBS="-L$dbdir -ldb4"
     fi
   elif test "$status" = "skip"; then
     svn_lib_berkeley_db=no
@@ -193,7 +193,7 @@
       ;;
       "std" )
         SVN_DB_INCLUDES=
-        SVN_DB_LIBS=-ldb
+        SVN_DB_LIBS=-ldb4
         svn_lib_berkeley_db=yes
       ;;
       *":"* )
@@ -201,13 +201,13 @@
 	lib="`echo $found | sed -e 's/^.*://'`"
         SVN_DB_INCLUDES="-I$header"
 dnl ### should look for a .la file
-        SVN_DB_LIBS="-L$lib -ldb"
+        SVN_DB_LIBS="-L$lib -ldb4"
         svn_lib_berkeley_db=yes
       ;;
       * )
         SVN_DB_INCLUDES="-I$found/include"
 dnl ### should look for a .la file
-        SVN_DB_LIBS="-L$found/lib -ldb"
+        SVN_DB_LIBS="-L$found/lib -ldb4"
 	svn_lib_berkeley_db=yes
       ;;
     esac
@@ -234,7 +234,7 @@
 AC_DEFUN(SVN_LIB_BERKELEY_DB_TRY,
   [
     svn_lib_berkeley_db_try_save_libs="$LIBS"
-    LIBS="$LIBS -ldb"
+    LIBS="$LIBS -ldb4"
 
     svn_check_berkeley_db_major=$1
     svn_check_berkeley_db_minor=$2
diff -urN subversion-20020412.org/ac-helpers/neon.m4 subversion-20020412/ac-helpers/neon.m4
--- subversion-20020412.org/ac-helpers/neon.m4	Fri Apr 12 20:50:00 2002
+++ subversion-20020412/ac-helpers/neon.m4	Fri Apr 12 20:51:06 2002
@@ -66,7 +66,7 @@
             args="--disable-shared"
           fi
   
-          SVN_SUBDIR_CONFIG(neon, $args --with-expat="$abs_srcdir/expat-lite/libexpat.la")
+          SVN_SUBDIR_CONFIG(neon, $args --with-expat="/usr/lib/libexpat.la")
 
           if test -f "$abs_builddir/neon/neon-config" ; then
             AC_MSG_CHECKING([for any extra libraries neon needs])
diff -urN subversion-20020412.org/build.conf subversion-20020412/build.conf
--- subversion-20020412.org/build.conf	Fri Apr 12 20:49:31 2002
+++ subversion-20020412/build.conf	Fri Apr 12 20:51:06 2002
@@ -65,7 +65,7 @@
 type = exe
 path = subversion/clients/cmdline
 libs = libsvn_client libsvn_wc libsvn_ra libsvn_delta libsvn_subr
-	$(SVN_APR_LIBS) libexpat
+	$(SVN_APR_LIBS) -lexpat
 manpages = subversion/clients/cmdline/man/svn.1
 infopages = doc/user/manual/svn-manual.info
  doc/user/svn_for_cvs_users/svn_for_cvs_users.info 
@@ -79,14 +79,14 @@
 type = exe
 path = subversion/svnadmin
 install = fs-bin
-libs = libsvn_repos libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_repos libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 # The subversion repository inspection tool
 [svnlook]
 type = exe
 path = subversion/svnlook
 install = fs-bin
-libs = libsvn_repos libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_repos libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 # Library needed by all subversion clients
 [libsvn_client]
@@ -143,7 +143,7 @@
 type = lib
 install = base-lib
 path = subversion/libsvn_subr
-libs =  $(SVN_APR_LIBS) libexpat
+libs =  $(SVN_APR_LIBS) -lexpat
 
 # Working copy management lib
 [libsvn_wc]
@@ -162,14 +162,6 @@
 custom = apache-mod
 install = apache-mod
 
-# Simple XML-parsing lib
-[libexpat]
-type = lib
-install = base-lib
-path = expat-lite
-# there are some .c files included by others, so *.c isn't appropriate
-sources = hashtable.c xmlparse.c xmlrole.c xmltok.c
-
 # ----------------------------------------------------------------------------
 #
 # TARGETS FOR SWIG SUPPORT
@@ -212,7 +204,7 @@
 sources = fs-test.c
 install = fs-test
 libs = libsvn_test libsvn_test_fs libsvn_fs libsvn_delta libsvn_subr
-	$(SVN_APR_LIBS) libexpat
+	$(SVN_APR_LIBS) -lexpat
 # run-fs-tests.sh will run this for us
 testing = skip
 
@@ -222,7 +214,7 @@
 path = subversion/tests/libsvn_fs
 sources = skel-test.c
 install = fs-test
-libs = libsvn_test libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 # run-fs-tests.sh will run this for us
 testing = skip
 
@@ -232,7 +224,7 @@
 path = subversion/tests/libsvn_fs
 sources = key-test.c
 install = fs-test
-libs = libsvn_test libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_fs libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 # run-fs-tests.sh will run this for us
 testing = skip
 
@@ -243,7 +235,7 @@
 sources = strings-reps-test.c
 install = fs-test
 libs = libsvn_test libsvn_test_fs libsvn_fs libsvn_delta libsvn_subr
-       $(SVN_APR_LIBS) libexpat
+       $(SVN_APR_LIBS) -lexpat
 # run-fs-tests.sh will run this for us
 testing = skip
 
@@ -253,7 +245,7 @@
 path = subversion/tests/libsvn_repos
 sources = md5args.c
 install = test
-libs = libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_subr $(SVN_APR_LIBS) -lexpat
 # run-repos-tests.sh will run this for us
 testing = skip
 
@@ -264,7 +256,7 @@
 sources = repos-test.c dir-delta-editor.c
 install = fs-test
 libs = libsvn_test libsvn_test_fs libsvn_repos libsvn_fs libsvn_delta
-	libsvn_subr $(SVN_APR_LIBS) libexpat
+	libsvn_subr $(SVN_APR_LIBS) -lexpat
 # run-repos-tests.sh will run this for us
 testing = skip
 
@@ -274,7 +266,7 @@
 path = subversion/tests/libsvn_subr
 sources = hashdump-test.c
 install = test
-libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 # test svn_stringbuf_t utilities
 [stringtest]
@@ -282,7 +274,7 @@
 path = subversion/tests/libsvn_subr
 sources = stringtest.c
 install = test
-libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 # test path library
 [path-test]
@@ -298,7 +290,7 @@
 path = subversion/tests/libsvn_wc
 sources = translate-test.c
 install = test
-libs = libsvn_test libsvn_delta libsvn_wc libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_delta libsvn_wc libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 # use random data to test delta processing
 [random-test]
@@ -306,7 +298,7 @@
 path = subversion/tests/libsvn_delta
 sources = random-test.c
 install = test
-libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 
 
 ### Tests that are simply broken (fix?)  ----------
@@ -317,7 +309,7 @@
 path = subversion/tests/libsvn_delta
 sources = delta-combine-test.c
 install = test
-libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 # disabled for now -- the delta combiner isn't written, and we
 # don't really have any idea if the interface would be compatible 
 # with this test, anyway.
@@ -330,7 +322,7 @@
 sources = ra-local-test.c
 install = fs-test
 libs = libsvn_test libsvn_ra libsvn_fs libsvn_delta libsvn_subr
-	$(SVN_APR_LIBS) libexpat
+	$(SVN_APR_LIBS) -lexpat
 # disabled for now, BROKEN.
 testing = skip
 
@@ -343,7 +335,7 @@
 path = subversion/tests/libsvn_subr
 sources = target-test.c
 install = test
-libs = libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_subr $(SVN_APR_LIBS) -lexpat
 # target-test.sh will run this for us
 testing = skip
 
@@ -353,7 +345,7 @@
 path = subversion/tests/libsvn_delta
 sources = xml-output-test.c
 install = test
-libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 # this is really redundant with our XML svn-test*.sh tests.
 testing = skip
 
@@ -363,7 +355,7 @@
 path = subversion/tests/libsvn_wc
 sources = checkout-test.c
 install = test
-libs = libsvn_wc libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_wc libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 # this is really redundant with our XML svn-test*.sh tests.
 testing = skip
 
@@ -375,7 +367,7 @@
 path = subversion/tests/libsvn_delta
 sources = deltaparse-test.c
 install = test
-libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_test libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 testing = skip
 
 
@@ -388,7 +380,7 @@
 path = subversion/tests/libsvn_delta
 sources = svndiff-test.c
 install = test
-libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 testing = skip
 
 # compare two files, print txdelta windows
@@ -397,7 +389,7 @@
 path = subversion/tests/libsvn_delta
 sources = vdelta-test.c
 install = test
-libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) libexpat
+libs = libsvn_delta libsvn_subr $(SVN_APR_LIBS) -lexpat
 testing = skip
 
 
