--- subversion-1.7.0/subversion/mod_dav_svn/mod_dav_svn.c.orig	2011-08-19 19:15:15.000000000 +0200
+++ subversion-1.7.0/subversion/mod_dav_svn/mod_dav_svn.c	2011-10-19 21:27:58.273487075 +0200
@@ -902,6 +902,7 @@
 /* Response handler for POST requests (protocol-v2 commits).  */
 static int dav_svn__handler(request_rec *r)
 {
+ if (dav_svn__get_root_dir(r)) {
   /* HTTP-defined Methods we handle */
   r->allowed = 0
     | (AP_METHOD_BIT << M_POST);
@@ -909,7 +910,7 @@
   if (r->method_number == M_POST) {
     return dav_svn__method_post(r);
   }
-
+ }
   return DECLINED;
 }
 
