diff -urN subversion-r1587.org/subversion/bindings/swig/python/setup.py subversion-r1587/subversion/bindings/swig/python/setup.py
--- subversion-r1587.org/subversion/bindings/swig/python/setup.py	Fri Apr 12 18:33:13 2002
+++ subversion-r1587/subversion/bindings/swig/python/setup.py	Fri Apr 12 18:38:11 2002
@@ -21,8 +21,8 @@
 from distutils.command import build_ext
 from distutils import dir_util
 
-INC_DIRS=['..', '../../../include', '../../../../apr/include']
-LIB_DIRS=['/usr/local/svn/lib']
+INC_DIRS=['..', '../../../include', '/usr/include/apache']
+LIB_DIRS=['/usr/lib','../../../libsvn_delta/.libs','../.libs','../../../libsvn_ra/.libs','../../../libsvn_subr/.libs']
 
 class build_swig(build_ext.build_ext):
   def initialize_options(self):
