diff -dur subversion-0.37.0.orig/tools/backup/hot-backup.py.in subversion-0.37.0/tools/backup/hot-backup.py.in
--- subversion-0.37.0.orig/tools/backup/hot-backup.py.in	2004-01-09 23:07:57.000000000 +0100
+++ subversion-0.37.0/tools/backup/hot-backup.py.in	2004-02-16 13:53:21.942581931 +0100
@@ -40,10 +40,14 @@
 # Command line arguments
 
 
-if len(sys.argv) != 3:
-  print "Usage: ", os.path.basename(sys.argv[0]), " <repos_path> <backup_path>"
+if not (sys.argv[1:2]==["-n"] and len(sys.argv)==5 or len(sys.argv)==3):
+  print "Usage: ", os.path.basename(sys.argv[0]), " [-n <num_backups>] <repos_path> <backup_path>"
   sys.exit(1)
 
+if sys.argv[1]=="-n":
+  num_backups=int(sys.argv[2])
+  sys.argv[1:]=sys.argv[3:]
+
 # Path to repository
 repo_dir = sys.argv[1]
 repo = os.path.basename(os.path.abspath(repo_dir))
