diff -urN subversion-1.4.0.org/build/ac-macros/neon.m4 subversion-1.4.0/build/ac-macros/neon.m4
--- subversion-1.4.0.org/build/ac-macros/neon.m4	2006-01-07 00:56:50.000000000 +0000
+++ subversion-1.4.0/build/ac-macros/neon.m4	2006-09-11 07:40:56.128959250 +0000
@@ -50,10 +50,8 @@
       NEON_VERSION=`cat $abs_srcdir/neon/.version`
       AC_MSG_RESULT([$NEON_VERSION])
 
-      if test -n "`echo \"$NEON_VERSION\" | grep '^0\.25\.'`" ; then
         AC_DEFINE_UNQUOTED([SVN_NEON_0_25], [1],
-                           [Defined if have Neon 0.25.x instead of 0.24.7.])
-      fi
+                           [Defined if have Neon 0.25.x/0.26.x instead of 0.24.7.])
 
       for svn_allowed_neon in $NEON_ALLOWED_LIST; do
         if test "$NEON_VERSION" = "$svn_allowed_neon" ||
@@ -123,10 +121,8 @@
       NEON_VERSION=`$neon_config --version | sed -e 's/^neon //'`
       AC_MSG_RESULT([$NEON_VERSION])
 
-      if test -n "`echo \"$NEON_VERSION\" | grep '^0\.25\.'`" ; then
         AC_DEFINE_UNQUOTED([SVN_NEON_0_25], [1],
-                           [Defined if have Neon 0.25.x instead of 0.24.7.])
-      fi
+                           [Defined if have Neon 0.25.x/0.26.x instead of 0.24.7.])
 
       for svn_allowed_neon in $NEON_ALLOWED_LIST; do
         if test "$NEON_VERSION" = "$svn_allowed_neon" ||
diff -urN subversion-1.4.0.org/configure.in subversion-1.4.0/configure.in
--- subversion-1.4.0.org/configure.in	2006-07-13 13:48:48.000000000 +0000
+++ subversion-1.4.0/configure.in	2006-09-11 07:37:59.757936750 +0000
@@ -69,7 +69,7 @@
 
 # Either a space-separated list of allowable Neon versions, or "any" to
 # mean allow anything.
-NEON_ALLOWED_LIST="0.24.7 0.25.0 0.25.1 0.25.2 0.25.3 0.25.4 0.25.5"
+NEON_ALLOWED_LIST="0.24.7 0.25.0 0.25.1 0.25.2 0.25.3 0.25.4 0.25.5 0.26.0 0.26.1 0.26.2"
 NEON_LATEST_WORKING_VER=0.25.5
 NEON_URL="http://www.webdav.org/neon/neon-${NEON_LATEST_WORKING_VER}.tar.gz"
 dnl You can skip the neon version check only if you know what you are doing 
