--- subversion/libsvn_subr/user.c~	2006-02-15 21:30:49.000000000 +0000
+++ subversion/libsvn_subr/user.c	2006-09-11 06:28:44.806268500 +0000
@@ -69,7 +69,7 @@
   const char *username;
   char *homedir;
 
-  if (apr_env_get(&homedir, "HOME", pool) == APR_SUCCESS)
+  if (apr_env_get(&homedir, "HOME_ETC", pool) == APR_SUCCESS || apr_env_get(&homedir, "HOME", pool) == APR_SUCCESS)
     return utf8_or_nothing(homedir, pool);
 
   username = get_os_username(pool);
